### PR TITLE
fix: side/nav bar style

### LINKED ISSUE #203: [Mobile Sidebar] Sidebar and Nav Bar Style Update
## 
1.Nav bar 高度為64
2.nav bar 的 pycon apac 字體改為 20 (放大）
-----------------------------------
1.sidebar 改為全屏
2.sidebar 保留上方 nav bar 內容，從右往左 move in （漢堡選單變為叉叉）
3.sidebar color is 121023
4.sidebar 裡的文字color is F0EBF5
5.sidebar dropdown color is 1A1A30
6.dropdown 字 color is E099E1
image
7.漢堡選單以及中英切換按鈕顏色為F0EBF5

https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=0%3A1

### PR DESCRIPTION
**Pull request by tim08094495757**
## Types of changes
* **Bugfix**
* **Refactoring**

## Description
1. fix #203
2. support active class for sponsor's link
3. make the order consistent with UI
4. allow the color to be changed by css

## Steps to Test This Pull Request
1. hover and click the topbar with both mobile and desktop version

## Expected behavior
1. 203
image
2. sponsor's link
image

## Related Issue
#203
## Additional context
I have questions about 
1. the selection between rem and px for this repo
2. the color on hover
image

### CODE REVIEW

File: components/core/header/Header.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <header class="w-full h-full">
+    <header class="w-full" :style="{ height: '64px' }">
```
Comment (mattwang44) [inline components/core/header/Header.vue:2]: One of the conventions in this project is avoiding inline style in the template. Please replace it with a newly-created class and add the CSS code for it in the `style` block.

File: components/core/header/Header.vue [hunk 2]
```diff
@@ -40,28 +39,17 @@ export default {
...
-    font-size: 19px;
-}
-
-.header-container.--desktop {
-    color: #c386ae;
+    font-size: 20px;
+    color: #f0ebf5;
     background-color: #121023;
 }
-
```
Comment (mattwang44) [inline components/core/header/Header.vue:50]: Please leave an empty line here.

File: components/core/header/nav-bar/NavBar.vue [hunk 3]
```diff
@@ -94,20 +94,25 @@ export default {
...
                 )
                 isOnCurrentPath =
                     isOnCurrentPath || paths.includes(this.$route.path)
             }
+            if (isLink) {
+                isOnCurrentPath =
+                    isOnCurrentPath ||
+                    this.$route.name === `${category}___${this.$i18n.locale}`
+            }
+            /*
```
Comment (mattwang44) [inline components/core/header/nav-bar/NavBar.vue:115]: How about deleting these unused lines?

**Review by mattwang44 — CHANGES_REQUESTED** [review]

File: components/core/header/Header.vue (hunk 1 above)
Comment (tim08094495757) [reply components/core/header/Header.vue]: https://github.com/pycontw/pycontw-frontend/pull/218/commits/1134d4e93beb0aeed8480dbb2ec357bafcc790d2

File: components/core/header/Header.vue (hunk 2 above)
Comment (tim08094495757) [reply components/core/header/Header.vue:50]: fixed: https://github.com/pycontw/pycontw-frontend/pull/218/commits/1134d4e93beb0aeed8480dbb2ec357bafcc790d2

**Comment by tim08094495757** [comment]
additional fix for the color on hover(confirmed with UI team): 
https://github.com/pycontw/pycontw-frontend/pull/218/commits/1191616dbfe037eb7171b0589946cfb12b3968c2

**Review by mattwang44 — APPROVED** [review]
Overall LGTM
but please remember to create a feature branch before any development and avoid using your `main` branch.

**Pull request merged**